apps/avatax/src/app/api/webhooks/checkout-calculate-taxes/route.ts

```diff
@@ -20,20 +22,28 @@ import { LogWriterFactory } from "@/modules/client-logs/log-writer-factory";
 import { AvataxInvalidAddressError } from "@/modules/taxes/tax-error";
 import { checkoutCalculateTaxesSyncWebhook } from "@/modules/webhooks/definitions/checkout-calculate-taxes";
 
+const logger = createLogger("checkoutCalculateTaxesSyncWebhook");
+
 const withMetadataCache = wrapWithMetadataCache(metadataCache);
 
 const checkoutCalculateTaxesSyncWebhookReponse =
   buildSyncWebhookResponsePayload<"CHECKOUT_CALCULATE_TAXES">;
 
 const handler = checkoutCalculateTaxesSyncWebhook.createHandler(async (_req, ctx) => {
+  after(async () => {
+    try {
+      await flushOtelMetrics();
+    } catch (error) {
+      logger.error("Error while flushing metrics", { error: error });
+    }
+  });
+
```

Comment:
probably this could be a middleware? so we wrap every handler and do this? `withOtelFlushing`?

Reply:
Added middleware in `withFlushOtelMetrics`